src/backend/base/langflow/schema/message.py

```
@@ -276,6 +278,13 @@ async def create(cls, **kwargs):
             return await asyncio.to_thread(cls, **kwargs)
         return cls(**kwargs)
 
+    def to_data(self, v: Any) -> Data:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)